main/ZsensorHTU21.ino

```diff
@@ -0,0 +1,124 @@
+/*  
+  OpenMQTTGateway Addon  - ESP8266 or Arduino program for home automation 
+ 
+   Act as a wifi or ethernet gateway between your 433mhz/infrared IR signal  and a MQTT broker
+   Send and receiving command by MQTT
+   
+   This is the Climate Addon:
+   - Measures Temperature, Humidity and Pressure
+   - Generates Values for: Temperature in degrees C and F, Humidity in %, Pressure in Pa, Altitude in Meter and Feet
+   - Required Hardware Module: HTU21
+   - Required Library: SparkFun HTU21 Library v1.1.3
+
+   Connection Schemata:
+   --------------------
+
+   HTU21 ------> Arduino Uno ----------> ESP8266
+   ==============================================
+   Vcc ---------> 5V -------------------> Vu (5V)
+   GND ---------> GND ------------------> GND
+   SCL ---------> Pin A5 ---------------> D1
+   SDA ---------> Pin A4 ---------------> D2
+ 
+    Copyright: (c) Hans-Juergen Dinges
```

Comment:
May you remove this old copyright please

Reply:
Removed copyright

Reply:
Thanks